lib/graphql/index.types.d.ts

```diff
@@ -0,0 +1,7 @@
+export type OperationError = {
+  message: string
+}
+
+export type LnInvoiceObject = {
```

Comment:
I think @UncleSamtoshi had some issues with putting the types in separate `*.d.ts` files?

Reply:
In his previous review, he was fine with it though. Can you be more specific?

Reply:
Its fine if this was the existing pattern, but I think tracking dependencies across modules can be clearer when types are manually imported instead of being accessible in the global scope which is what ```.d.ts``` does.